concepticondata/references/references.bib

```diff
@@ -1,3 +1,14 @@
+@dataset{Sidwell2021,
```

Comment:
I prefer not to use the Zenodo BibTEx Directly. Rather, we have already an adjusted  bibtex entry in our lexibank dataset, in sources, haven't we? I think we even discussed this, so @MuffinLinwist, if you could check that and use this instead. If it is the same, we discuss here how to represent the data consistently as a `@book` in BibTex.

Reply:
In our lexibank dataset, I've put the same reference. I checked with the Vietic dataset and it seems to me that is the same but the only difference is the entry `@book` instead of `@dataset`. So, I'm changing this (in the lexibank dataset too) and waiting for further comments.